oaknational/Oak-Web-Application
# Feat/lesq 378/form links in new tab [LESQ-378]

## Description

- Update `CopyrightNotice` component to accept a parameter which will toggle whether links open in a new tab or not 
- Add tests for this new param
- Update privacy policy on downloads page to open in a new tab

## How to test

1. Go to https://deploy-preview-2014--oak-web-application.netlify.thenational.academy/programmes/science-primary-ks2/units/earth-sun-and-moon/lessons/the-shape-of-earth/downloads?preselected=all
2. You should see links within the form have an external icon and open in a new tab

## Screenshots

How it used to look (delete if n/a):
<img width="489" alt="Screenshot 2023-10-12 at 13 45 43" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/990f118c-86a3-4c7f-b72c-b275f6b868c3">


How it should now look:
<img width="493" alt="Screenshot 2023-10-12 at 13 45 14" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/a8c06016-b40b-44a8-bde8-1c9d408f738f">

## ACs

- [ ]  The OGL and Terms and conditions, privacy policy links should open in a new tab only on the downloads page
    - They provide more information in the middle of a form, we don’t want to interrupt a user completing a form by navigating them to a new page in their current tab
- [ ]  Links should warn the user they are being taken to a new tab (icon and aria)
